ci: Failed to build packages on CentOS 7 due to epel repo

For now there's something wrong with get metalink from the epel repository in the CI running on CentOS 7, please see https://github.com/api7/apisix-build-tools/pull/86/checks?check_run_id=3468216581 for details.

```
+ rpm -ivh epel-release-latest-7.noarch.rpm
warning: epel-release-latest-7.noarch.rpm: Header V4 RSA/SHA256 Signature, key ID 352c64e5: NOKEY
Preparing...                          ########################################

Updating / installing...

epel-release-7-13                     ########################################

+ yum install -y yum-utils readline-dev readline-devel

Loaded plugins: fastestmirror, ovl

Loading mirror speeds from cached hostfile
 One of the configured repositories failed (Unknown),

 and yum doesn't have enough cached data to continue. At this point the only

 safe thing yum can do is fail. There are a few ways to work "fix" this:
     1. Contact the upstream for the repository and get them to fix the problem.
     2. Reconfigure the baseurl/etc. for the repository, to point to a working
        upstream. This is most often useful if you are using a newer

        distribution release than is supported by the repository (and the

        packages for the previous distribution release still work).
     3. Run the command with the repository temporarily disabled
            yum --disablerepo=<repoid> ...
     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:
            yum-config-manager --disable <repoid>
        or
            subscription-manager repos --disable=<repoid>
     5. Configure the failing repository to be skipped, if it is unavailable.
        Note that yum will try to contact the repo. when it runs most commands,
        so will have to try and fail each time (and thus. yum will be be much
        slower). If it is a very temporary problem though, this is often a nice
        compromise:
            yum-config-manager --save --setopt=<repoid>.skip_if_unavailable=true
Cannot retrieve metalink for repository: epel/x86_64. Please verify its path and try again

The command '/bin/sh -c /install-common.sh install_apisix_dependencies_rpm' returned a non-zero code: 1

make: *** [Makefile:74: build-apisix-rpm] Error 1
Error: Process completed with exit code 2.
```

--- a/utils/install-common.sh
+++ b/utils/install-common.sh
@@ -19,6 +19,8 @@ install_dependencies_rpm() {
     yum -y install wget tar gcc automake autoconf libtool make curl git which unzip
     wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
     rpm -ivh epel-release-latest-7.noarch.rpm
+    # Fix failed to get metalink from epel repository
+    yum --disablerepo=epel -y update ca-certificates
     yum install -y yum-utils readline-dev readline-devel
 
     # install lua 5.1 for compatible with openresty 1.17.8.2

